libs/blocks/caas-marquee-metadata/caas-marquee-metadata.js

```diff
@@ -23,7 +23,7 @@ function getUrl(ctaLink) {
 }
 function parseCtas(el) {
   const ctas = {};
-  const ctaLinks = el.querySelectorAll('a');
+  const ctaLinks = el.querySelectorAll('a') || [];
```

Comment:
@cmiqueo This was noticed by James in the dev channel:
This line should check for null/undefined since the || '' below it won't capture those cases. 
const ctaLinks = el?.querySelectorAll('a') || [];

Reply:
@fullcolorcoder 
`el?.querySelectorAll('a')` will never return NULL or UNDEFINED
if no 'a' are found it will return an empty array:
```
document.querySelectorAll('aaaa')
NodeList []
```

Reply:
The change has been made, anyway.

Reply:
I see that now. Thank you for the quick responses :)